library/private/options_tools.h

```diff
@@ -191,6 +194,46 @@ ratio_t parse(const std::string& str)
 template<>
 color_t parse(const std::string& str)
 {
+  std::string s = options_tools::trim(str);
+  std::transform(s.begin(), s.end(), s.begin(), [](unsigned char c) { return std::tolower(c); });
```

Comment:
```suggestion
```
do we need to lowercase the whole string if we're going to regex match with `icase` anyways?